C7525: inline variables require at least '/std:c++17' in win32_helpers.h

I am not sure whether this is intended, but as I could not find any note about C++ 17 being a requirement and the header containing other `__WI_LIBCPP_STD_VER` guards, there might be a missing `#ifdef` guard in win32_helpers.h around

```c++
inline constexpr weak_ordering weak_ordering::less{static_cast<signed char>(-1)};
inline constexpr weak_ordering weak_ordering::equivalent{static_cast<signed char>(0)};
inline constexpr weak_ordering weak_ordering::greater{static_cast<signed char>(1)};
```

If I consume this header, for instance via wil/com.h, in a VS2022 project using the C++ language standard "Default (ISO C++ 14 Standard), I get 

> error C7525: inline variables require at least '/std:c++17'

--- a/include/wil/win32_helpers.h
+++ b/include/wil/win32_helpers.h
@@ -130,9 +130,9 @@ struct weak_ordering
     signed char m_value;
 };
 
-inline constexpr weak_ordering weak_ordering::less{static_cast<signed char>(-1)};
-inline constexpr weak_ordering weak_ordering::equivalent{static_cast<signed char>(0)};
-inline constexpr weak_ordering weak_ordering::greater{static_cast<signed char>(1)};
+__WI_LIBCPP_INLINE_VAR constexpr weak_ordering weak_ordering::less{static_cast<signed char>(-1)};
+__WI_LIBCPP_INLINE_VAR constexpr weak_ordering weak_ordering::equivalent{static_cast<signed char>(0)};
+__WI_LIBCPP_INLINE_VAR constexpr weak_ordering weak_ordering::greater{static_cast<signed char>(1)};
 
 #endif
 } // namespace wistd

